src/main/java/org/openpnp/model/AbstractVisionSettings.java

```diff
@@ -0,0 +1,65 @@
+package org.openpnp.model;
+
+import org.openpnp.spi.VisionSettings;
+import org.openpnp.vision.pipeline.CvPipeline;
+import org.simpleframework.xml.Attribute;
+import org.simpleframework.xml.Element;
+
+public abstract class AbstractVisionSettings extends AbstractModelObject implements VisionSettings {
+    @Attribute()
```

Comment:
Should this implement the interface `Named` and make` getName()` an `@Override`?

Reply:
Good point, I didn't noticed the Named interface. Will add it.